.github/renovate.json5

```
@@ -275,6 +275,25 @@
         '/github\\.com/fluent/.+/',
       ],
     },
+    // Add `distroless-` prefix to tag when updating `envoyproxy`.
+    {
+      matchDatasources: [
+        'github-releases'
+      ],
+      matchPackageNames: [
+        'github.com/envoyproxy/envoy'
```

It's always a bit fuzzy trying to understand what the actual package name for Renovate is without checking the logs, but wouldn't it be just `envoyproxy/envoy` for this one (even though it's coming from GitHub releases)? I'm basing this off the PR title and PR body that Renovate generated here: https://github.com/gardener/gardener/pull/12882
```suggestion
        'envoyproxy/envoy'
```

---

You are right, I was confused by the `sourceRepository` field in `containers.yaml` 😅  But `github.com/` is removed by the custom manager ([ref](https://github.com/gardener/ci-infra/blob/69a8d200e5ba22b98550b89c73e2ac7948902d2a/config/renovate/imagevector.json5#L16)).